### DISCUSSION TITLE
Example code to test media storage?

### DISCUSSION THREAD

**Discussion by pamelafox** [cookiecutter/cookiecutter-django#4039] [Q&A] [answered]
Hi there! From what I can tell, the default app UI doesn't have a media upload interface. I would like to test out the media storage mechanism as I'm on the Azure team at Microsoft and want to make sure that Azure storage works well. Does anyone have example code that utilizes the media storage? If not, my plan is to add an ImageField to the user model/form, but I wanted to check first. Thanks!

**Comment by browniebroke** [comment, marked as answer]
> From what I can tell, the default app UI doesn't have a media upload interface. 

That is correct, the default project doesn't have any filefield by default. 

> I would like to test out the media storage mechanism as I'm on the Azure team at Microsoft and want to make sure that Azure storage works well. 

Cool! Thanks for helping out, especially since I'm not using Azure myself. The feature was contributed by the community recently. 

> Does anyone have example code that utilizes the media storage? If not, my plan is to add an ImageField to the user model/form

Yes, that should be a good minimal working example. You should be able to upload from the admin if you don't want to touch the front-end part.

**Reply by pamelafox** [reply]
Thanks, I'll give it a go!

**Comment by pamelafox** [comment]
In case anyone runs into this thread and wants to do something similar... 

Here's the commit where I added the ImageField:

https://github.com/pamelafox/cookiecutter-django-output/commit/217dda71cc536eb1c7d60dc6a8cb9c792dd1f320

Azure Storage worked fine, with a container configured for Public read access to blobs. Here's the Bicep configuration for the storage:

https://github.com/pamelafox/cookiecutter-django-output/blob/bc185c606ee27a54d86aef98ecdec0c4dc60a313/infra/resources.bicep#L90
